RELEASE.md

```diff
@@ -1,3 +1,43 @@
+# Release 2.0.0
+
+The 2.0 minor series tracks TensorFlow 2.0.
+
+## Breaking changes
+
+- TensorBoard now serves on localhost only by default to avoid unintentional
+  overexposure. To expose TensorBoard to the network, either use a proxy, bind
+  to a specific hostname or IP address by using the `--host` flag, or explicitly
+  enable the previous behavior of binding on all network interfaces by passing
+  the flag `--bind_all`. See PR #2589.
+
+- The `--logdir` flag no longer supports passing multiple comma-delimited paths,
+  which means that it now *supports* paths containing literal comma and colon
+  characters, like `./logs/m=10,n=20,lr=0.001` or `./logs/run_12:30:15`. If you
+  need the old behavior (which only some plugins support), you can use the
+  separate flag `--logdir_spec` instead. See PR #2664.
```

Comment:
Might want to mention symlink trees here so people don’t think that they
need the old behavior when they actually don’t.

Reply:
Done.